frontends/PyCDE/src/pycde/value.py

```diff
@@ -296,9 +296,10 @@ def create(*args, **kwargs):
           if isinstance(created, support.NamedValueOpView):
             created = created.opview
 
-          # Return a Value.
-          assert len(created.results) == 1
-          return Value(created.results[0])
+          # Return the wrapped values, if any.
+          converted_results = tuple(Value(res) for res in created.results)
+          return converted_results[0] if len(
```

Comment:
Nit: why not avoid creating the tuple in the case when len == 1?

Reply:
Allows for a bit more consistent (imo) syntax when binding to the unpacked ops:


e.g.:
```python
def retTuple(n):
  return (x for x in range(n))

a = retTuple(1)
print(f"a = {a}")
a, b = retTuple(2)
print(f"a = {a}, b = {b}")
```
prints
```
a = <generator object retTuple.<locals>.<genexpr> at 0x7f2f989e8ba0>
a = 0, b = 1
```